Escape single quotes(') and right single quotation mark(’) in the file path

xGreenNarae opened the pull request:
## Overview
This Pull Request addresses an issue where file paths containing `single quotes (')` and `right single quotation mark(’ == u+2019)` cause errors in the `getWindowsShortcutProperties` function.

## Problem Description
The `generateCommands` function in the `get-windows-shortcut-properties` package constructs a PowerShell command using file paths. If a file path includes single quotes or (’), it breaks the command syntax in PowerShell, leading to execution failures.

## Solution
I have updated the `generateCommands` function to escape single quotes in file paths. This is accomplished by replacing each single quote or (’) in the file path with two single quotes ('') or (’’), for Powershell syntax.

xGreenNarae committed:
Escape Single quotes in the file path

TheJaredWilcurt commented on index.js:
```diff
@@ -23,17 +23,19 @@ function throwError (customLogger, message, error) {
 }
 
-function generateCommands (filePaths, customLogger) {
+function generateCommands(filePaths, customLogger) {
```
When calling/executing a function, the parenthesis should always touch the function name. When defining a function, the parenthesis should never touch the function name. Doing this consistently allows for easier visual parsing/scanning of code.
```suggestion
function generateCommands (filePaths, customLogger) {
```

xGreenNarae replied:
Oh, I've never thought about this before. I think it's a very desirable method!

TheJaredWilcurt commented on index.js:
```diff
@@ -23,17 +23,19 @@ function throwError (customLogger, message, error) {
   for (let filePath of filePaths) {
     const normalizedFile = normalizeFile(filePath, customLogger);
     if (normalizedFile) {
+      // Escape single quotes in the file path
```
> I have updated the generateCommands function to escape single quotes in file paths. This is accomplished by replacing each single quote in the file path with two single quotes (''), which is the standard method for escaping single quotes in PowerShell.

What you have explained here is the important part. Explain that in the comment in a clear/concise way.

TheJaredWilcurt reviewed (changes requested):
For 3 years I have requested GitHub not require me to type in a comment into the "request changes" box if I had already added inline comments on a PR, and for 3 years GitHub has failed to change this stupid requirement.

xGreenNarae committed:
Escape right single quotation mark(u+2019 == ’)

xGreenNarae committed:
Improve comment and function declaration

xGreenNarae commented:
I realized that not only single quotes, but also u+2019 (') characters called right single quotation mark also cause errors, and I added this along with the content of your review and corrected it!

TheJaredWilcurt commented:
Merged, new release published to npm as `get-windows-shortcut-properties@1.2.0`. Thanks for the PR.
